src/data/faq_de.json

```diff
@@ -102,6 +102,159 @@
+                            "Zu Beginn des Jahres wurde durch den Europäischen Rat beschlossen, einen interoperablen und standardisierten Impfnachweis für medizinische Zwecke auf den Weg zu bringen. Deutschland setzt diese europäische Entscheidung nun schnell um, um allen Bürgern, die dies wünschen, einen digitalen Impfnachweis zur Verfügung stellen zu können. Mittlerweile hat die Europäische Kommission auch einen Verordnungsentwurf zur Regelung eines EU-weiten Anerkennungsrahmens für (digitale) Zertifikate für Impfungen, Tests und für Personen mit COVID-19-Heilstatus vorgelegt, um die Reisefreizügigkeit zu erleichtern. Das grüne Zertifikat der EU bildet den Rechtsrahmen für die Lösungen der Mitgliedsstaaten."
+                        ]
+                    },
+                    {
+                        "title": "Wer kann einen elektronischen Impfnachweis erhalten? Wie erhalte ich ein Impfzertifikat? Wie kann ich, wenn ich bereits vollständigen Impfschutz haben, den Status nachträglich in die Corona-Warn-App übertragen?",
+                        "anchor": "vaccination_certificate_who",
+                        "active": true,
+                        "textblock": [
+                            "Alle Personen, die in einen Anspruch auf eine Impfung haben, können sich ergänzend zum analogen Impfnachweis auch einen digitalen Impfnachweis ausstellen lassen. <br> Sie können gegebenenfalls ein Corona-Impfzertifikat von der Arztpraxis oder dem Impfzentrum erhalten, wo die Impfung vorgenommen wurde. Fragen Sie vor Ort ganz einfach nach. Die Impfzentren und die Arztpraxen sind jedoch nicht dazu verpflichtet, Impfzertifikate auszustellen. Zukünftig wird die Ausstellung eines Impfzertifikats auch in Apotheken möglich sein. <br>Es werden verschiedene Möglichkeiten geprüft, um auch nachträglich digitale Impfnachweise für Corona-Impfungen zu erstellen. Grundsätzlich soll die nachträgliche Ausstellung dort erfolgen, wo man geimpft worden ist. Wenn in den Impfzentren entsprechende Kontaktdaten vorliegen, sollen die 2D-Barcodes (QR-Codes) möglichst automatisch per Post zugesandt werden. Außerdem könnten Ärztinnen und Ärzte sowie Apothekerinnen und Apotheker nachträglich Impfnachweise ausstellen."
+                        ]
+                    },
+                    {
+                        "title": "Kann ein vollständiger Impfstatus in der App als Berechtigung zum Reisen dienen?",
+                        "anchor": "vaccination_certificate_privileges_travel",
+                        "active": true,
+                        "textblock": [
+                            "Ob Sie mit einem vollständigen Impfschutz beispielsweise reisen dürfen, hängt von den Bestimmungen des jeweiligen Landes ab. COMMENT fynnngodau: Maybe we could find a resource which provides further information here? -> ToDo, probably the best place is to look at the webpage of the ministry of foreign affairs of the corresponding countries. Maybe suggest this?"

```

Comment:
If there are still to-dos in the PR, then it would be better to put the PR in draft status.

https://www.auswaertiges-amt.de/de/ReiseUndSicherheit/reise-und-sicherheitshinweise
I think this is only available in German.

Reply:
yep, sorry about that!